certified-connectors/DQ on Demand/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7276 @@
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Input"
+          },
+          {
+            "in": "query",
+            "name": "characterType",
+            "x-example": "LowerCaseConsonant",
+            "enum": [
+              "Digit",
+              "Letter",
+              "LetterOrDigit",
+              "Punctuation",
+              "Whitespace",
+              "Upper",
+              "Lower",
+              "Symbol",
+              "UpperCaseVowel",
+              "LowerCaseVowel",
+              "UpperCaseConsonant",
+              "LowerCaseConsonant",
+              "OddDigit",
+              "EvenDigit",
+              "NonPrinting"
+            ],
+            "required": true,
+            "type": "string",
+            "description": "The type of characters to remove.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Character Type"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Request.",
+            "schema": {
+              "$ref": "#/definitions/DQGlobal"
+            },
+            "examples": {
+              "application/json": {
+                "status": 200,
+                "message": "success",
+                "data": "Sae Te Iu"
+              }
+            }
+          }
+        }
+      }
+    },
+    "/StringExtension/RemoveSingleCharacterWords": {
+      "x-summary": "RemoveSingleCharacterWords",
+      "get": {
+        "operationId": "StringExtRemoveSingleWords",
+        "summary": "Remove Single Character Words",
+        
+        "description": "Remove all single characters from the input I.E. `This is a test` -> `This is test`",
+        "parameters": [
+          {
+            "in": "query",
+            "name": "Input",
+            "x-example": "pull together a sentence.",
+            "required": true,
+            "type": "string",
+            "description": "The value to alter.",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "Input"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Request.",
+            "schema": {
+              "$ref": "#/definitions/DQGlobal"
+            },
+            "examples": {
+              "application/json": {
+                "status": 200,
+                "message": "success",
+                "data": "pull together sentence."
+              }
+            }
+          }
+        }
+      }
+    },
+    "/StringExtension/ReplaceAdjacentRepeatingText": {
+      "x-summary": "ReplaceAdjacentRepeatingText",
+      "get": {
+        "operationId": "StringExtReplaceRepeatingText",
+        "summary": "Replace Adjacent Repeating Text",
+        "description": "Check and replace repeated neighbouring characters.",
+        
+        "parameters": [
+          {
+            "in": "query",
+            "name": "Input",
+            "x-example": "exaaaaaaaaaample",
```

Comment:
@lukeswanton If you do not have a valid example to provide for an operation, please the example sections, these are not required.